### PR TITLE
Add option to specify the MySQL port for the Joomla importers

### PR DESCRIPTION
**Pull request by ienev**
* Add option to specify the MySQL port for the Joomla importers
* Use Ruby 1.9 hash syntax
* Update the Joomla importers documentation

### CODE REVIEW

**Comment by ienev** [comment]
@parkr I'll fix the conflicts, but are OK with this PR?

**Comment by ienev** [comment]
Conflicts solved.

File: lib/jekyll-import/importers/joomla.rb [hunk 1]
```diff
@@ -32,11 +33,12 @@ def self.process(options)
         dbname  = options.fetch('dbname')
         user    = options.fetch('user')
         pass    = options.fetch('password', '')
-        host    = options.fetch('host', "localhost")
+        host    = options.fetch('host', 'localhost')
+        port    = options.fetch('port', '3306')
```
Comment (parkr) [inline lib/jekyll-import/importers/joomla.rb]: Isn't this port supposed to be an integer?

Comment (ienev) [inline lib/jekyll-import/importers/joomla.rb]: Hmm, looks [like](https://github.com/brianmario/mysql2/blob/master/lib/mysql2/client.rb#L85) [it](https://github.com/brianmario/mysql2/blob/master/ext/mysql2/client.c#L78). Should I add `to_i` or change the default value... or both?

Comment (parkr) [inline lib/jekyll-import/importers/joomla.rb]: Both! And please update the documentation so it doesn't have the quotes.

**Comment by ienev** [comment]
One of the build jobs seems stuck. Is this normal, should I just wait or there is a way to restart the jobs?

**Comment by parkr** [comment]
No, that's not normal! I tried to restart it but it didn't work. Weird! Merging anyway.

@jekyllbot: merge

**Pull request merged**